napari/util/io.py

```diff
@@ -32,6 +34,10 @@ def magic_imread(filenames, *, use_dask=None, stack=True):
     image : array-like
         Array or list of images
     """
+    # cast Path to string
```

Comment:
Can you edit the doc string of `filenames` in `magic_imread` to indicate `Path` objects are now accepted